aspnetcore/security/authentication/identity-api-authorization.md

```diff
@@ -202,7 +202,9 @@ Some web clients might not include cookies in the header by default:
 
 ## Use token-based authentication
 
-For clients that don't support cookies, the login API provides a parameter to request tokens. A custom token (one that is proprietary to the ASP.NET Core identity platform) is issued that can be used to authenticate subsequent requests. The token is passed in the `Authorization` header as a bearer token. A refresh token is also provided. This token allows the application to request a new token when the old one expires without forcing the user to log in again.
+For clients that don't support cookies, the login API provides a parameter to request tokens, but you must be careful not to leak tokens. We recommend using cookies in browser-based applications, because the browser will automatically handle them without exposing them to JavaScript by default.
```

Comment:
```suggestion
For clients that don't support cookies, the login API provides a parameter to request tokens, but you must be careful not to leak tokens. We recommend using cookies in browser-based applications, because the browser automatically handles them without exposing them to JavaScript by default.
```

Reply:
Same sort of forcefulness as above, and swap around where by default lands. 

We recommend using cookies in browser-based applications, because, by default the browser automatically handles them without exposing them to JavaScript.

Reply:
```suggestion
We recommend using cookies in browser-based applications, because, by default, the browser automatically handles them without exposing them to JavaScript.
```